corehq/apps/reports/formdetails/readable.py

```diff
@@ -48,7 +50,12 @@ def get_questions(domain, app_id, xmlns):
             _("Remote apps are not supported")
         )
 
-    xform = app.get_xform_by_xmlns(xmlns)
+    try:
+        xform = app.get_xform_by_xmlns(xmlns)
+    except ResourceNotFound:
+        raise QuestionListNotFound(_(
```

Comment:
Should the exception be notified so that developers are aware that there is something weird happening with the system? Otherwise this will fail silently

Reply:
:+1: That's a very good idea.